trunk/SUAVE/Components/Energy/Converters/Rotor.py

```diff
@@ -146,13 +147,16 @@ def spin(self,conditions):
         """         
            
         #Unpack    
-        B       = self.number_blades
+        B       = self.number_of_blades
+        E       = self.number_of_engines
```

Comment:
same comment, this breaks with the object oriented nature of SUAVE.

Reply:
There was a lot of inconsistency (some "number_engines", some "number_of_engines"  so I made all number_of_enginges.